### PR TITLE
NR-115593 MEVN quickstart

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
# Summary

**Ticket details**: https://issues.newrelic.com/browse/NR-115593
**Description**: This PR has the new quickstart integration named "MEVN"

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/mevn/config.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Title of the quickstart
+title: MEVN Stack
+
+summary: |
+  Monitor your apps using New Relic MEVN integration with Vue, MongoDB and NodeJS agents for browser, database and application monitoring.
+
+# Description of the quickstart
+description: |
+  ## Comprehensive MEVN Stack Monitoring System
+  MEVN stack is a popular Javascript software stack that has become very popular in recent times to build powerful web applications. It comprises four essential components, namely MongoDB, ExpressJS, Vue, and NodeJS. These components integrate seamlessly and logically, making it easy to create web applications using this powerful technology stack. Each of the four components has a crucial role to play in the development of web applications.
```
Comment (mdumpati) [inline quickstarts/mevn/config.yml:14]:
```suggestion
  MEVN stack is a popular Javascript software stack that has become very popular in recent times to build powerful web applications. It comprises four essential components, namely MongoDB, ExpressJS, Vue, and NodeJS. These components integrate seamlessly and logically, making it easy to create web applications using this powerful technology stack. Each of these four components has a crucial role to play in the development of web applications.
```

File: quickstarts/mevn/config.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+  ## What should you look for in a MEVN dashboard?
+  - MongoDB metrics such as active connections, database details, and usage time
+  - For ExpressJS and NodeJS, there are a variety of metrics available, including web transactions, top ten errors, and memory usage
```
Comment (mdumpati) [inline quickstarts/mevn/config.yml:18]:
```suggestion
  - For ExpressJS and NodeJS, there are a variety of metrics available, including web transactions, the top ten errors, and memory usage
```

File: quickstarts/mevn/config.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  - Vue metrics include viewed urls, page views by city, ajax requests, etc. In addition, you may create your own chart based on the metrics collected
```
Comment (mdumpati) [inline quickstarts/mevn/config.yml:19]:
```suggestion
  - Vue metrics include viewed urls, page views by city, AJAX requests, etc. In addition, you may create your own chart based on the metrics collected
```

File: quickstarts/mevn/config.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+  ## What’s included in the MEVN quickstart?
+  Install this quickstart to setup preconfigured observability solutions:
+  - A number of high-value alerts are available including the average number of connections created, the FCP score and the memory usage
```
Comment (mdumpati) [inline quickstarts/mevn/config.yml:23]:
```suggestion
  - A number of high-value alerts are available including the average number of connections created, the FCP score and memory usage
```

**Review by mdumpati — COMMENTED** [review]
@RamanaReddy8801 check my comments

File: quickstarts/mevn/config.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  ## What should you look for in a MEVN dashboard?
+  - MongoDB metrics such as active connections, database details, and usage time
+  - For ExpressJS and NodeJS, there are a variety of metrics available, including web transactions, the top ten errors, and memory usage
+  - Vue metrics include viewed urls, page views by city, AJAX requests, etc. In addition, you may create your own chart based on the metrics collected
+
+  ## What’s included in the MEVN quickstart?
+  Install this quickstart to setup preconfigured observability solutions:
+  - A number of high-value alerts are available including the average number of connections created, the FCP score and memory usage
+  - Informative dashboards (the top 10 transactions, connections, database indexes, and more)
+
```
Comment (pkosanam) [inline quickstarts/mevn/config.yml:25]:
```suggestion
  - MongoDB metrics such as active connections, database details, and usage time
  - For ExpressJS and NodeJS, there are a variety of metrics available, including web transactions, the top ten errors, and memory usage
  - Vue metrics include viewed urls, page views by city, AJAX requests, etc. In addition, you may create your own chart based on the metrics collected
```

File: quickstarts/mevn/config.yml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+# Description of the quickstart
+description: |
+  ## Comprehensive MEVN Stack Monitoring System
+  MEVN stack is a popular Javascript software stack that has become very popular in recent times to build powerful web applications. It comprises four essential components, namely MongoDB, ExpressJS, Vue, and NodeJS. These components integrate seamlessly and logically, making it easy to create web applications using this powerful technology stack. Each of these four components has a crucial role to play in the development of web applications.
+
+  ## What should you look for in a MEVN dashboard?
+  - MongoDB metrics such as active connections, database details, and usage time
+  - For ExpressJS and NodeJS, there are a variety of metrics available, including web transactions, the top ten errors, and memory usage
+  - Vue metrics include viewed urls, page views by city, AJAX requests, etc. In addition, you may create your own chart based on the metrics collected
+
```
Comment (pkosanam) [inline quickstarts/mevn/config.yml:20]: can remove lines 16 to 19

File: quickstarts/mevn/config.yml [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  ## What’s included in the MEVN quickstart?
+  Install this quickstart to setup preconfigured observability solutions:
+  - A number of high-value alerts are available including the average number of connections created, the FCP score and memory usage
+  - Informative dashboards (the top 10 transactions, connections, database indexes, and more)
```
Comment (pkosanam) [inline quickstarts/mevn/config.yml:24]: can remove line 24

File: quickstarts/mevn/config.yml [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+  MEVN stack is a popular Javascript software stack that has become very popular in recent times to build powerful web applications. It comprises four essential components, namely MongoDB, ExpressJS, Vue, and NodeJS. These components integrate seamlessly and logically, making it easy to create web applications using this powerful technology stack. Each of these four components has a crucial role to play in the development of web applications.
+
+  ## What should you look for in a MEVN dashboard?
+  - MongoDB metrics such as active connections, database details, and usage time
+  - For ExpressJS and NodeJS, there are a variety of metrics available, including web transactions, the top ten errors, and memory usage
+  - Vue metrics include viewed urls, page views by city, AJAX requests, etc. In addition, you may create your own chart based on the metrics collected
+
+  ## What’s included in the MEVN quickstart?
+  Install this quickstart to setup preconfigured observability solutions:
+  - A number of high-value alerts are available including the average number of connections created, the FCP score and memory usage
```
Comment (pkosanam) [inline quickstarts/mevn/config.yml:23]: line 23 will be mentioned at the last after all the bullet points

**Review by pkosanam — CHANGES_REQUESTED** [review]

**Review by seemantk — APPROVED** [review]
lgtm

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by pkosanam — APPROVED** [review]
LGTM

**Pull request merged**